### PR TITLE
Fix footer

### PR DESCRIPTION
**Pull request by krishguptadev**
## Related Issue 

- Information about the related issue 

Closes: # 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

## Code of Conduct

- [ ] I agree to follow this project's [Code of Conduct](https://github.com/web3community/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +12,10 @@
     "@snowpack/plugin-dotenv": "^2.2.0",
     "@snowpack/plugin-optimize": "^0.2.13",
     "astro": "^0.20.12",
+    "autoprefixer": "^10.4.0",
     "common-tags": "^1.8.0",
     "postcss": "^8.3.10",
+    "sass": "^1.43.4",
```
Comment (krishguptadev) [inline package.json:18]:
```suggestion
```

**Review by KukretiShubham — APPROVED** [review]
LGTM 🚀

**Review by shuklaritvik06 — APPROVED** [review]
LGTM 🎉

**Review by vinzvinci — COMMENTED** [review]
LGTM 👍

**Review by Panquesito7 — COMMENTED** [review]
Amazing! 🚀

**Pull request merged**